tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Orders GetAll 4826.feature

```diff
@@ -4,42 +4,42 @@
-    And the orders list is returned with the following values
-        | OrderId    | Description         | Status      | Created    | LastUpdated | LastUpdatedBy                        |
-        | C000014-01 | Some Description    | Submitted   | 11/05/2020 | 11/05/2020  | 335392e4-4bb1-413b-9de5-36a85c9c0422 |
-        | C000014-02 | Another Description | Unsubmitted | 05/05/2020 | 09/05/2020  | a11a46f9-ce6f-448a-95c2-fde6e61c804a |
+	When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
+	Then a response with status code 200 is returned
+	And the orders list is returned with the following values
+		| OrderId    | Description         | Status      | Created    | LastUpdated | LastUpdatedByName |
+		| C000014-01 | Some Description    | Submitted   | 11/05/2020 | 11/05/2020  | Bob Smith         |
+		| C000014-02 | Another Description | Unsubmitted | 05/05/2020 | 09/05/2020  | Alice Smith       |
 
 @4826
 Scenario: 2. Get all of the orders from an invalid organisationId
-    When a GET request is made for a list of orders with organisationId 3a72c6ab-0be8-4faa-8cb0-3b2c1f077eeb
-    Then a response with status code 200 is returned
-    And an empty list is returned
+	When a GET request is made for a list of orders with organisationId 3a72c6ab-0be8-4faa-8cb0-3b2c1f077eeb
+	Then a response with status code 200 is returned
+	And an empty list is returned
 
 @4826
 Scenario: 3. If a user is not authorised then they cannot access the orders
-    Given no user is logged in
-    When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
-    Then a response with status code 401 is returned
+	Given no user is logged in
+	When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
+	Then a response with status code 401 is returned
 
 @4826
 Scenario: 4. A non buyer user cannot access the orders
-    Given the user is logged in with the Authority role for organisation 4af62b99-638c-4247-875e-965239cd0c48
-    When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
-    Then a response with status code 403 is returned
+	Given the user is logged in with the Authority role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+	When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
+	Then a response with status code 403 is returned
 
 @4826
 Scenario: 5. Service Failure
-    Given the call to the database will fail
-    When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
-    Then a response with status code 500 is returned
+	Given the call to the database will fail
+	When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
+	Then a response with status code 500 is returned
```

Comment:
I thought we discussed in code standards meetings or something like that to have a newline at the end of a feature file

Reply:
👍